src/main/java/org/terasology/blockNetwork/TwoNetworkNodes.java

```diff
@@ -39,6 +55,12 @@ public boolean equals(Object o) {
                 || that.node1.equals(node2) && that.node2.equals(node1);
     }
 
+    /**
+    * hashCode constructor.
```

Comment:
This isn't a constructor. Write the description of what the function does instead.